README.md

```diff
@@ -204,26 +209,43 @@ public IEnumerator VisualTestStub_Generate()
 
 ### Making a manual build
 
-1. Build unity WASM with its name as `unity` into the folder `/static`. **It's very important that the folder/build name is `unity`**.
-2. Checkout the file named `static/unity/Build/DCLUnityLoader.js`. Unity deletes anything on this folder as part of the build process and we need that.
+#### Option 1
+1. Run `make watch` in a terminal at `kernel/static`. Wait for it to finish and leave it watching for changes. **This step must be done before replacing the build files or they may be replaced again with the latest published build files when the `make watch` process runs**
```

Comment:
`make watch` shouldn't be done at directory `kernel/static` I find this kind of misleading

Reply:
totally XD

Reply:
Done